test/src/models/form_group_test.dart

```diff
@@ -107,7 +107,7 @@ void main() {
     test('Reset group restores value of all controls to null', () {
       final formGroup = FormGroup({
         'name': FormControl(value: 'john doe'),
-        'email': FormControl(value: 'johndoe@reactiveforms.com'),
+        'email': FormControl<String>(value: 'johndoe@reactiveforms.com'),
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**